res/controllers/Pioneer DDJ-200.midi.xml

```diff
@@ -0,0 +1,526 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset schemaVersion="1" mixxxVersion="2.0.0+">
+    <info>
+        <name>Pioneer DDJ-200</name>
+        <author>Dan Giddins</author>
+        <description>Pioneer DDJ-200 configuration for 2 decks.</description>
+    </info>
+    <controller id="DDJ-200">
+        <scriptfiles>
+            <file filename="Pioneer-DDJ-200-scripts.js" functionprefix="DDJ200"/>
+        </scriptfiles>
+        <controls>
+            <control>
+                <group>[EqualizerRack1_[Channel1]_Effect1]</group>
+                <key>parameter3</key>
+                <description>MIDI Learned from 364 messages.</description>
```

Comment:
These descriptions are just noise without any value and should be removed from the final version.

Reply:
Descriptions removed.